insights/v1alpha1/types_insights.go

```diff
@@ -171,7 +171,7 @@ type GathererStatus struct {
 	// lastGatherDuration represents the time spent gathering.
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Type=string
-	// +kubebuilder:validation:Pattern="^([1-9][0-9]*(\\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$"
+	// +kubebuilder:validation:Pattern="^(([0-9]+(?:\\.[0-9]+)?(ns|us|μs|ms|s|m|h))+)$"
```

Comment:
`μs` is highlighting as a change, are we sure there isn't a difference in unicodes here?